### PR TITLE
EWPP-544: Render file patterns only when file is set.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: src/DocumentMediaValueExtractor.php [hunk 1]
```diff
@@ -77,6 +77,10 @@ protected static function getFromRemoteFile(MediaInterface $media): ?FileValueOb
     }
 
     $file_link = $media->get('oe_media_remote_file')->first();
+    if ($media->get('oe_media_remote_file')->first()->getSize() == 0) {
```
Comment (sergepavle) [inline src/DocumentMediaValueExtractor.php]: ===

Comment (sergepavle) [reply src/DocumentMediaValueExtractor.php]: We can reuse $file_link variable.

File: templates/content/media--document--default.html.twig [hunk 2]
```diff
@@ -6,8 +6,10 @@
  * @see ./core/themes/stable/templates/content/media.html.twig
  */
 #}
-{{ pattern('file_translation', {
-  'button_label': 'Download'|t,
-  'file': file|default([]),
-  'translations': translations|default([]),
-}) }}
+{% if file is not empty %}
```
Comment (sergepavle) [inline templates/content/media--document--default.html.twig]: We can drop `is not empty`. Everything else will be handled by twig.

**Review by sergepavle — CHANGES_REQUESTED** [review]
I've left a few comments.

**Review by sergepavle — DISMISSED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by upchuk — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**